### DISCUSSION TITLE
Extend the Snapshot Mechanism

### DISCUSSION THREAD

**Discussion by janhartje** [directus/directus#15964] [Archive]
We are using Multiple Instances of Directus, for our different Environments. Currently we are using the Snapshot mechanism to transport the Colletions between our Environments. It would be nice to have the same function for Flows and Permissions.

**Comment by rijkvanzanten** [comment]
I believe this is a duplicate of https://github.com/directus/directus/discussions/15921 🙂

**Reply by janhartje** [reply]
@rijkvanzanten yeah that would solve my problem too. Thank you :)
